pyflask/manageNeuroconv/manage_neuroconv.py

```diff
@@ -69,6 +73,35 @@ def get_metadata_schema(source_data: Dict[str, dict]) -> Dict[str, dict]:
     return json.loads(json.dumps(dict(results=metadata, schema=schema), cls=NWBMetaDataEncoder))
 
 
+class objectview(object):
+    def __init__(self, d):
+        self.__dict__ = d
+
+
+def validate_metadata(parent: dict, function: str) -> dict:
+    """
+    Function used to validate data using an arbitrary NWB Inspector function
+    """
+    fn = nwbinspector.__dict__.get(function)
```

Comment:
```suggestion
    check_function: callable = nwbinspector.__dict__.get(check_function_name)
```
I might make a mapping like this in the Inspector sometime - technically there is the `available_checks`, but it is a list so you'd have to do search instead of direct reference like this